man/systemd.netdev.xml

```diff
@@ -1848,6 +1848,28 @@
     Linux Ethernet Bonding Driver HOWTO</ulink></para>
   </refsect1>
 
+  <refsect1>
+    <title>[Xfrm] Section Options</title>
+
+    <para>The <literal>[Xfrm]</literal> section accepts the following
```

Comment:
The table further up with the kinds of network devices we support also needs an entry